classpath/java/lang/System.java

```diff
@@ -114,9 +110,11 @@ private static Properties makeProperties() {
   public static native long currentTimeMillis();
 
   public static native int identityHashCode(Object o);
+  
+  private static native long getBaseNanoTime();
```

Comment:
its in the pull request.  Its not a file just for it, but its a method the java-lang.cpp file.